src/Network/Probe.php

```diff
@@ -688,22 +686,21 @@ private static function detect(string $uri, string $network, int $uid, array $ap
 		}
 
 		$parts = parse_url($uri);
-
-		if (empty($parts['scheme']) || !empty($parts['host']) && strstr($uri, '@')) {
-			// If the URI starts with "mailto:" then jump directly to the mail detection
-			if (strpos($uri, 'mailto:') !== false) {
-				$uri = str_replace('mailto:', '', $uri);
-				return self::mail($uri, $uid);
-			}
-
-			if ($network == Protocol::MAIL) {
-				return self::mail($uri, $uid);
-			}
-		} else {
+		if (empty($parts['scheme']) && empty($parts['host']) && !strstr($parts['path'], '@')) {
```

Comment:
Wouldn't this be invoked with an URL in the format `mailto:user@domain.tld`?

Reply:
```php
$parsed = parse_url('mailto:user@domain.tld');
// array(2) {
//   ["scheme"]=>
//   string(6) "mailto"
//   ["path"]=>
//   string(15) "user@domain.tld"
// }
```